src/views/Alteration.vue

```diff
@@ -281,7 +270,7 @@ export default class Alteration extends Vue {
       // initialize Fee Summary data
       const filingData = [this.alterationResource.filingData]
       filingData.forEach(fd => {
-        fd.futureEffective = this.getEffectiveDateTime.isFutureEffective
+        (fd as FilingDataIF).futureEffective = this.getEffectiveDateTime.isFutureEffective
```

Comment:
Another build type error, now fixed.

(As with the others, VS Code / Volar / eslint didn't complain about this, and in fact, hovering over the properties gave the correct types. I dunno.)